tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/generator/op/OpGenerator.java

```diff
@@ -335,15 +335,15 @@ private void createApiDef(OpDef opDef, File apiDefFile) throws IOException {
             visibility = ApiDef.Visibility.SKIP;
             apiDef.setVisibility(visibility);
             break;
-          case "D":
-          case "d":
+          case "B":
+          case "b":
```

Comment:
Why did this option get renamed to `b`? It's still documented as `d` on line 319.

Reply:
Oops, that was a left over, reverted it